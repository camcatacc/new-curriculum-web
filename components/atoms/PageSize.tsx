import React from "react";

export const PageSize = (props: { children: any }) => {
    return (
        <div className="w-screen max-w-screen-xl pl-8 pr-8 md:pl-16 md:pr-16">
            {props.children}
        </div>
    );
};
