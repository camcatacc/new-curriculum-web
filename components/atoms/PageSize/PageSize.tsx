// Modules
import React from "react";

// Definitions
export interface PageSizeProps {
	children: React.ReactNode;
	className?: string;
}

// Element
const PageSize = ({ children, className }: PageSizeProps) => <div className={`${className} w-full max-w-screen-xl pl-4 pr-4 md:pl-24 md:pr-24`}>{children}</div>;

export default PageSize;
