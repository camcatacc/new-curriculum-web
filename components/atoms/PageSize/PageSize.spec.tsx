// Modules
import React from "react";
import { render } from "@testing-library/react";

// Elements
import PageSize from "components/atoms/PageSize/PageSize";

// Definitions
import type { PageSizeProps } from "components/atoms/PageSize/PageSize";

// Default PRops
const defaultProps: PageSizeProps = {
	children: <p>Bu</p>
};

// Tests
describe("PageSize", () => {
	it("Renders as expected", () => {
		const { container } = render(<PageSize {...defaultProps} />);
		expect(container).toMatchSnapshot();
	});
});
