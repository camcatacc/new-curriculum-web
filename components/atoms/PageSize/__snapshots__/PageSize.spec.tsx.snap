// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSize Renders as expected 1`] = `
<div>
  <div
    class="undefined w-full max-w-screen-xl pl-4 pr-4 md:pl-24 md:pr-24"
  >
    <p>
      Bu
    </p>
  </div>
</div>
`;
